### ISSUE TITLE
Flesh out content if someone answers NO to having a CU

### ISSUE THREAD

**Issue by ErnieAtLYD** [codeforamerica/mdc-step-wizard#35]
Currently says: "A Certificate of Use will be issued to you as part of the short-term event permitting process."

Sophia, let me know if you need me to give you content for this.

**Comment by phiden** [comment]
@ErnieAtLYD Yes please to content for this, ASAP but no later than 7/16! See also issue #43

**Comment by ErnieAtLYD** [comment]
Okay, I'm looking at my own notes. This is what it says:

Let's try the following content, feel free to edit where necessary:
- If you do not have a CU, you will need to apply for a “Special” CU. 
- [Fill the Certificate of Use Application](http://www.miamidade.gov/permits/library/certificate-of-use.pdf). The following are required when filling out the CU form: 
  - The name and address of the business. 
  - The square footage of the area to be occupied. 
  - The type of business and business owners' name. If the business is a corporation, the name of a corporate officer is required. 
  - The folio number for the property is also necessary.
- The CU applications must be delivered by hand to the PIC. 
  A non-refundable up front fee of $111.24 which includes the 8% surcharge is required upon submittal of the Certificate of Use application.

I got that from this [older version of the doc](https://docs.google.com/document/d/1COloNUx0Kfmj6JeRgAIXEzGg96I8-S-DzRIMVVXiztE/edit) I wrote. As for why, I looked at some older notes from Donna: "As the process currently exists, the Zoning plan Reviewer does not issue the zoning approval on the short term event until the CU is issued." She writes about how the process needs to be streamlined, but it's more of a note to self as opposed to what actually goes on at the moment.
